scripts/module/ghostcars.nut

```diff
@@ -0,0 +1,40 @@
+function onScriptLoad() {
+  GHOSTKEY <- BindKey( true, 0x47, 0, 0 ); // G
+  ghost <- array( 100, false );
+}
+
+function onKeyDown( player, id ) {
+  if (id == GHOSTKEY) {
+    ghost[player.ID] = !ghost[player.ID];
+    local message = "Your vehicle is a ghost: ";
+    if (ghost[player.ID]) {
+      if (player.VehicleSlot == 0) { // Player is driver
+        player.Vehicle.IsGhost = true;
+      }
+      message += "ON";
+    } else {
+      if (player.VehicleSlot == 0) {
+        player.Vehicle.IsGhost = false;
+      }
+      message += "OFF";
+    }
+    MessagePlayer(message, player);
+  }
+}
+
+function onPlayerCommand( player, cmd, text ) {
+  cmd = cmd.tolower();
+  if (cmd == "help") {
+    MessagePlayer("[#8080A0][G] - Ghostcar", player);
+  }
+}
+
+function onPlayerEnterVehicle( player, vehicle, door ) {
+  if (ghost[player.ID] && door == 1) {
+    vehicle.IsGhost = true;
+  }
+}
+
+function onPlayerExitVehicle( player, vehicle ) {
+  vehicle.IsGhost = false;
```

Comment:
Wouldn't this also happen if a passenger left your car?

Reply:
Hmm, yeah, probably. I'll add a check for `vehicle.Driver` - should be `== null` if the driver is the one leaving. Or `== player.ID`.